examples/instance/docker/instance_test.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/docker/docker/client"
+	"github.com/docker/infrakit/pkg/spi/instance"
+	"github.com/docker/infrakit/pkg/types"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	testNamespace = map[string]string{"cluster": "test", "type": "testing"}
+	tags          = map[string]string{"group": "workers"}
+)
+
+func TestInstanceLifecycle(t *testing.T) {
```

Comment:
CircleCI can't really run tests with docker reliably or at all.  Instead of removing this test, do this....

1. `import testutil "github.com/docker/infrakit/pkg/testing"`
2. Add a check here:
```
	if testutil.SkipTests("docker") {
		t.SkipNow()
	}
```
This will skip the test on CircleCI because there's an environment variable set when running tests on CircleCI (see the `circle.yml` file):
```
    SKIP_TESTS: "docker,etcd"
```